### ISSUE TITLE
Instance segmentatiokn model stopped at 95 epoches

### ISSUE THREAD

**Issue by Shamsu992** [ultralytics/hub#1161]
### Search before asking

- [x] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

<img width="2654" height="1309" alt="Image" src="https://github.com/user-attachments/assets/40cc6dd2-ab8c-4264-8199-6f4081982015" />

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Shamsu992, thank you for raising an issue about [Ultralytics HUB](https://hub.ultralytics.com) 🚀! Please visit our [HUB Docs](https://docs.ultralytics.com/hub) to learn more:
- [**Quickstart**](https://docs.ultralytics.com/hub/quickstart). Start training and deploying YOLO models with HUB in seconds.
- [**Datasets: Preparing and Uploading**](https://docs.ultralytics.com/hub/datasets). Learn how to prepare and upload your datasets to HUB in YOLO format.
- [**Projects: Creating and Managing**](https://docs.ultralytics.com/hub/projects). Group your models into projects for improved organization.
- [**Models: Training and Exporting**](https://docs.ultralytics.com/hub/models). Train YOLOv5 and YOLOv8 models on your custom datasets and export them to various formats for deployment.
- [**Integrations**](https://docs.ultralytics.com/hub/integrations). Explore different integration options for your trained models, such as TensorFlow, ONNX, OpenVINO, CoreML, and PaddlePaddle.
- [**Ultralytics HUB App**](https://docs.ultralytics.com/hub/app). Learn about the Ultralytics App for iOS and Android, which allows you to run models directly on your mobile device.
    * [**iOS**](https://docs.ultralytics.com/hub/app/ios). Learn about YOLO CoreML models accelerated on Apple's Neural Engine on iPhones and iPads.
    * [**Android**](https://docs.ultralytics.com/hub/app/android). Explore TFLite acceleration on mobile devices.
- [**Inference API**](https://docs.ultralytics.com/hub/inference-api). Understand how to use the Inference API for running your trained models in the cloud to generate predictions.
If this is a 🐛 Bug Report, please provide screenshots and steps to [reproduce](https://docs.ultralytics.com/help/minimum_reproducible_example/) your problem to help us get started working on a fix.
If this is a ❓ Question, please provide as much information as possible, including dataset, model, environment details etc. so that we might provide the most helpful response.
This is an automated response, and an Ultralytics engineer will also assist you soon. Thank you for your patience!

**Comment by yogendrasinghx** [comment]
Hi @Shamsu992,

Thanks for reporting this issue. Could you please share the model URL or model ID so we can investigate it further?

**Comment by Shamsu992** [comment]
Sure here you go: https://hub.ultralytics.com/models/icASO0yrZ8Ei6n8HZWNh

**Comment by pderrenger** [comment]
Hi @Shamsu992! 👋

Thank you for providing the model URL - this will help us investigate the issue where your instance segmentation model stopped at 95 epochs.

There are several potential reasons why training might have stopped early:

## Common Causes:

1. **Early Stopping**: Ultralytics HUB uses early stopping by default to prevent overfitting. If your model's validation performance stops improving for a certain number of epochs (typically 50), training will automatically halt. This is actually a feature to save compute resources and prevent overfitting.

2. **Resource Limits**: Cloud training sessions have time and resource limits that might have been reached.

3. **Configuration Issues**: There might be specific settings in your training configuration that caused the stop.

## Next Steps:

Could you please check the following:

1. **Training Logs**: In your model page, check the training logs or console output for any messages about why training stopped. Look for mentions of "EarlyStopping" or error messages.

2. **Validation Metrics**: Review your validation loss and mAP curves - if they plateaued or started increasing, early stopping likely triggered.

3. **Training Configuration**: What were your training settings (epochs, patience, etc.)?

If this appears to be unexpected behavior rather than intentional early stopping, please share:
- Any error messages from the training logs
- Your training configuration details
- Whether the model was still improving when it stopped

This will help us determine if there's an actual issue that needs to be addressed. Thanks for your patience as we investigate! 🔍

**Comment by yogendrasinghx** [comment]
Hi @Shamsu992,

Thanks for sharing the model URL. I investigated your model and found that the dataset used has an incorrect local absolute path in the yaml file:

```
path: 'C:/Users/shams/Downloads/Dates-seg-v1'
train: images/train
val: images/val
```


As per the [official Ultralytics HUB dataset example](https://github.com/ultralytics/hub/blob/b94701388634c746b844f89839a81a6b7ce1c96e/example_datasets/coco8-seg/coco8-seg.yaml#L11), the correct format should be:

```
path:
train: images/train
val: images/val
```

Please reupload your dataset using this correct format and start a fresh training.

I’ve also refunded the credits used for this training. Apologies for the inconvenience caused  and thank you for your helpful feedback and bug reports that help us continue to improve the product.
